examples/kuka_iiwa_arm/BUILD.bazel

```diff
@@ -0,0 +1,122 @@
+        "@drake//systems/primitives:discrete_derivative",
+        "@drake//manipulation/kuka_iiwa:kuka_iiwa",
+        "@drake//systems/lcm:lcm_interface_system",
+        ":kuka_torque_controller",
+    ],
+)
+
+
+cc_binary(
+    name="iiwa_oscillate",
+    srcs = ["iiwa_oscillate.cc"],
+    deps = [
+            "//systems/controllers:endeffector_velocity_controller",
+            "//systems/controllers:endeffector_position_controller",
+            "//systems/controllers:safe_velocity_controller",
+            "@drake//systems/analysis:simulator",
+            "@drake//systems/framework",
+            "@drake//systems/primitives:multiplexer",
+            "@drake//systems/primitives:constant_vector_source",
+            "@drake//systems/primitives:sine",
+            "@drake//lcm",
+            "@drake//examples/kuka_iiwa_arm:iiwa_common",
+            "@drake//examples/kuka_iiwa_arm:iiwa_lcm",
+            "@drake//common",
+    ]
+)
+
+cc_binary(
+    name = "iiwa_visualizer",
+    srcs = ["iiwa_visualizer.cc"],
+    data = [
+        "@drake//examples/kuka_iiwa_arm:models",
+        "@drake//manipulation/models/iiwa_description:models",
+    ],
+    deps = [
+            "//common:common",
+            "@drake//examples/kuka_iiwa_arm:iiwa_common",
+            "@drake//examples/kuka_iiwa_arm:iiwa_lcm",
+            "@drake//examples/kuka_iiwa_arm:kuka_torque_controller",
+            "@drake//attic/manipulation/util:sim_diagram_builder",
+            "@drake//attic/multibody:rigid_body_tree_construction",
+            "@drake//attic/multibody/rigid_body_plant:frame_visualizer",
+            "@drake//attic/systems/controllers:rbt_inverse_dynamics_controller",
+            "@drake//common:text_logging_gflags",
+            "@drake//lcm",
+            "@drake//systems/analysis",
+            "@drake//systems/controllers:state_feedback_controller_interface",
+            "@drake//systems/primitives:discrete_derivative"
+    ]
+)
+
+cc_library(
+    name = "kuka_torque_controller",
+    srcs = ["kuka_torque_controller.cc"],
+    hdrs = ["kuka_torque_controller.h"],
+    visibility = ["//visibility:public"],
+    deps = [
+        "@drake//multibody/plant",
+        "@drake//systems/controllers:inverse_dynamics",
+        "@drake//systems/controllers:pid_controller",
+        "@drake//systems/controllers:state_feedback_controller_interface",
+        "@drake//systems/framework:diagram_builder",
+        "@drake//systems/framework:leaf_system",
+        "@drake//systems/primitives:adder",
+        "@drake//systems/primitives:pass_through",
+        "@drake//systems/primitives:constant_vector_source",
+    ],
+)
+
+cc_binary(
+    name="iiwa_controller_demo",
+    srcs = ["iiwa_controller_demo.cc"],
+    deps = [
+            "//systems/controllers:endeffector_velocity_controller",
+            "//systems/controllers:endeffector_position_controller",
+            "@drake//common/trajectories:piecewise_polynomial",
```

Comment:
Here and elsewhere. For everything that isn't a unit test, you should use only one drake dependency (to support pythong binding).

`        "@drake//:drake_shared_library",`